scripts/findDesignSystemVersions/README.md

```diff
@@ -0,0 +1,39 @@
+# Find Design System Versions Tool
+
+This tool gathers the versions of our design systems currently being used the CMS-WDS organization on github.cms.gov.
+
+You can run this tool from the root of the repository directory with the following command:
+
+```
+yarn scan-versions
+```
+
+## Setup
+
+First, you'll need to create a personal access token through GitHub Enterprise.
```

Comment:
Would it make more sense to [link to the Github docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for this in the instance they update the process/UI for creating a token?

Reply:
Doh! Yes, good idea